docs/src/pages/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -40,6 +40,7 @@ import TextField from '@material-ui/core/TextField';
 instead of top level imports (without a Babel plugin):
 
 ```js
+// 🐢 Slow
```

Comment:
Well, it could be OK, the problem is in development during the initial bootstrap. It's slower, but slow?